config.yml.example

```diff
@@ -1,8 +1,28 @@
+# This is the configuration file for Icinga DB.
+
 database:
   host: icingadb
   port: 3306
   database: icingadb
   user: icingadb
   password: icingadb
+
 redis:
   address: redis:6380
+
+logging:
+  # Default logging level. Can be set to 'fatal', 'error', 'warning', 'info' or 'debug'.
+  # If not set, defaults to 'info'.
+  level:
+
+  # Map of component-logging level pairs to define a different log level than the default value for each component.
```

Comment:
Do we want to duplicate our config docs here?

Reply:
Why not?

Reply:
For the same reason why you've removed the example config from the docs. Because... it's a duplicate.

Reply:
So you want to remove the comments? If so, I'm sorry, no.

Reply:
Because of https://github.com/Icinga/icingadb/pull/366#discussion_r717630965 ?

Reply:
I don't want to look into the docs. So there are some comments.